app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,46 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
`stylesheet_link_tag` is for Sprockets css and `stylesheet_pack_tag` is for Webpacker css, but why do we need both for css, one is enough isn't is?

Reply:
I actually don't know. 🤔 
I would've asked you. haha

Reply:
Seems like will produce inconsistent outcomes in dev and prod, check in [webpacker issues](https://github.com/rails/webpacker/issues/2342), but seems like don't have really concrete explanation

Reply:
According to this [webpacker pr #2608](https://github.com/rails/webpacker/pull/2608), it is a problem with different config between production and development in `webpacker.yml`. 

**Changes needed to me made**
1. Change `extract css: false` to `extract css: true` in `webpacker.yml` - maintain outcome consistency in dev and prod
2. Remove `stylesheet_link_tag` in `application.html.erb` - since we have the `stylesheet/application.scss` in `app/javascript`, we are using Webpacker, so use only `stylesheet_pack_tag`

Reply:
so, does this solve the issue?

Reply:
Since we are not using Sprockets, do we need to delete `app/assets/stylesheets/application.css`?

Reply:
> so, does this solve the issue?

yes, solved

Reply:
I am not sure. Can you trry?

Reply:
It does not have any problem after deleting

Reply:
ok. if no problem then can resolve already right?

Reply:
ya, later, I will push again and ask for review